tad/services/tasks.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from collections.abc import Sequence
+from typing import Annotated
+
+from fastapi import Depends
+
+from tad.models.task import Task
+from tad.models.user import User
+from tad.repositories.tasks import TasksRepository
+from tad.services.statuses import StatusesService
+
+logger = logging.getLogger(__name__)
+
+
+class TasksService:
+    def __init__(
+        self,
+        statuses_service: Annotated[StatusesService, Depends(StatusesService)],
+        repository: Annotated[TasksRepository, Depends(TasksRepository)],
+    ):
+        self.repository = repository
+        self.statuses_service = statuses_service
+
+    def get_tasks(self, status_id: int) -> Sequence[Task]:
+        return self.repository.find_by_status_id(status_id)
+
+    def assign_task(self, task: Task, user: User) -> Task:
+        task.user_id = user.id
+        return self.repository.save(task)
+
+    def move_task(
+        self, task_id: int, status_id: int, previous_sibling_id: int | None = None, next_sibling_id: int | None = None
+    ) -> Task:
+        """
```

Comment:
I prefer not using the docs part unless you want to make something extra clear. The code should be readable by itself.